docs/getting-started/development-environment.rst

```diff
@@ -178,7 +178,105 @@ images are having problems building.
 Linux
 ~~~~~
 
-Want to write it? Contact us on the chat!
+Useful packages to install:
+
+.. code-block:: sh
+
+        git:      $ apt install git-all
+
+        zoom:     https://support.zoom.us/hc/en-us/articles/204206269-Installing-or-updating-Zoom-on-Linux#h_adcc0b66-b2f4-468b-bc7a-12c182f354b7
```

Comment:
I would remove Zoom and Discord, they are more for CERN users and our work, not really for global Invenio users :)